homeassistant/components/homematicip_cloud/light.py

```diff
@@ -117,3 +124,148 @@ def supported_features(self):
     async def async_turn_off(self, **kwargs):
         """Turn the light off."""
         await self._device.set_dim_level(0)
+
+
+class HomematicipNotificationLight(HomematicipGenericDevice, Light):
+    """Representation of HomematicIP Cloud dimmer light device."""
+
+    _channel_index = None
+
+    _channel_name = None
+
+    # Dictionary to translate between RGBColorState and hs_color
+    _color_switcher = None
+
+    def __init__(self, home, device, channel_name):
+        """Initialize the dimmer light device."""
+        super().__init__(home, device)
```

Comment:
```
def __init__(self, home, device, channel_index):
         """Initialize the dimmer light device."""
        if channel_index == 1:
           super().__init__(home, device, 'Top') 
       else:
           super().__init__(home, device, 'Bottom') 
```
The 3rd parameter will be added to the name property at the end already in the base class.

Reply:
Now i understood your first comment from yesterday. I was on a complete wrong way.
Reverted some code and now i use the post param.

Thanks a lot